src/bcs/MFEMScalarFunctorDirichletBC.C

```diff
@@ -0,0 +1,27 @@
+#include "MFEMScalarFunctorDirichletBC.h"
+
+registerMooseObject("PlatypusApp", MFEMScalarFunctorDirichletBC);
+
+InputParameters
+MFEMScalarFunctorDirichletBC::validParams()
+{
+  InputParameters params = MFEMEssentialBC::validParams();
+  params.addRequiredParam<platypus::MFEMScalarCoefficientName>(
+      "coefficient", "The coefficient setting the values on the essential boundary.");
```

Comment:
[FunctorDirichletBC](https://mooseframework.inl.gov/source/bcs/FunctorDirichletBC.html) in MOOSE refers to this as `functor`, so we should probably do the same here (and elsewhere in the code).

Reply:
Done this now.